util/src/main/java/com/google/cloud/hadoop/util/AbstractGoogleAsyncWriteChannel.java

```diff
@@ -128,17 +128,11 @@ public void handleResponse(S response) throws IOException {}
    * may override this method to return the expected "identifier" response. Return null to let the
    * exception propagate through correctly.
    */
-  public S createResponseFromException(IOException ioe) {
+  public S createResponseFromException(IOException e) {
     return null;
   }
 
-  @Deprecated
-  public void setUploadBufferSize(int bufferSize) {
-    setUploadChunkSize(bufferSize);
-  }
-
-  /** Sets size of upload buffer used. */
-  public void setUploadChunkSize(int chunkSize) {
+  private static int validateUploadChunkSize(int chunkSize) {
```

Comment:
Maybe getUploadChunkSize with validate logic inside? A method that begins with validate seems to return true/false.

Reply:
Yeah, agree this is confusing, but I want to move it inside `AsyncWriteChannelOptions` in follow up CL.